scripts/pipeline/populate_new_database.pl

```diff
@@ -276,27 +277,59 @@ =head2 OLD DATA
 $skipped_mlss_ids{$_->dbID} = 1 for grep {$_->has_tag($rerun_tag)} @$all_default_method_link_species_sets;
 
 if($only_show_intentions) {
-    print "GenomeDB entries to be copied:\n";
+    my $out;
+    $out .= "GenomeDB entries to be copied:\n";
```

Comment:
I would suggest to remove the legacy output altogether, as it duplicates the JSON output, and would be harder to work with than the JSON output.
```suggestion
```